biztalk/core/sftp-adapter.md

```diff
@@ -12,7 +12,10 @@ ms.topic: "article"
 BizTalk Server includes an **SFTP** adapter to send and receive messages from a secure FTP server using the SSH file transfer protocol. This topic includes the steps to configure an **SFTP** receive location, and configure an SFTP send port to receive and send messages from a secure FTP server. It also includes common questions and answers.
 
 ## Prerequisites
-**Starting with BizTalk Server 2016**, the SFTP adapter uses WinSCP to connect to SFTP, and therefore supports a larger range of SFTP servers. **Download [WinSCP](http://winscp.net)** on the BizTalk Server runtime. Be sure to check the supported WinSCP versions for each BizTalk Server verison:
+**Starting with BizTalk Server 2016**, the SFTP adapter uses WinSCP to connect to SFTP, and therefore supports a larger range of SFTP servers. **Download [WinSCP](http://winscp.net)** on the BizTalk Server runtime. Be sure to check the default supported WinSCP versions for each BizTalk Server version:
+ * BizTalk Server 2020 with CU6 - WinSCP version 6.3.3
```

Comment:
We can wait to release this change until CU6 is live or we can update up to CU5 which is missing and do one more change later once CU6. I am just afraid it will be forgotten again if we don't do it now.